holoviews/links.py

```diff
@@ -0,0 +1,45 @@
+from collections import defaultdict
```

Comment:
Couple of suggestions where this might live:

* Maybe ``hv.util.links``?
* Or maybe ``hv.plotting.links`` (this might be better actually)

Reply:
Let's go with ``hv.plotting.links``.